CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Spatial filtering: add a new checkbox "Filter by user" to filter by users and not by user groups
```

Comment:
Add a note for LWC 3.6

Reply:
NB: this will also work with LWC 3.5. I will add a comment in the `CHANGELOG.md`